sycl/doc/extensions/experimental/sycl_ext_oneapi_bindless_images.asciidoc

```diff
@@ -1342,8 +1348,9 @@ void write_image(unsampled_image_handle ImageHandle,
 Inside a kernel, it's possible to retrieve data from an image via `fetch_image` 
 or `sample_image`, passing the appropriate image handle. The `fetch_image` API 
 is applicable to sampled and unsampled images, and the data will be fetched 
-exactly as is in device memory. The `sample_image` API is only applicable to 
-sampled images, the image data will be sampled according to the 
+exactly as is in device memory.
```

Comment:
The formatting here and elsewhere is different than the rest of the document. With each sentence starting on their own line and not continuing from the last sentence, breaking when reaching the character limit.

Would be good to align the formatting with the rest of the document.

Reply:
I see, thanks I'll fix this.

Reply:
Done